### PR TITLE
Fix how docker image is executed in the OCI Compute Instance

### PR DESCRIPTION
**Pull request by jhorvath**
Fix and improvement how docker image is executed in the OCI Compute Instance.

### CODE REVIEW

File: java/java.lsp.server/vscode/src/extension.ts [hunk 1]
```diff
@@ -1098,13 +1108,37 @@ async function runDockerSSH(username: string, host: string, dockerImage: string,
         micronautConfigFilesEnv += `${bootstrapProperties ? "," : ""}${applicationPropertiesContainerPath}`;
     } 
 
-    let dockerPullCommand = "";
+    let script = `#!/bin/sh\n`;
```
[the remark this answers is not in the document]
Comment (jhorvath) [reply java/java.lsp.server/vscode/src/extension.ts:1111]: I was considering that but finally decided keep the fix simple

**Review by sdedic — APPROVED** [review]

**Comment by neilcsmith-net** [comment]
@jhorvath seriously?! https://cwiki.apache.org/confluence/display/NETBEANS/Pull+requests+for+delivery

**Comment by mbien** [comment]
@jhorvath since this is the third PR you merged without taking the agreed delivery rules into account and/or replying to the pings, it would be good if you could clarify what is going on here.

**Comment by sdedic** [comment]
sorry; emails lost in spam. I have to apologize -- I overlooked that the changes target `delivery` branch during review, focused just on the code changes; otherwise I would raise the question :-/ Sorry, I'll try to do better next time.

**Pull request merged**